modules/social_features/social_embed/social_embed.module

```diff
@@ -41,3 +42,113 @@ function social_embed_create_media_button() {
   $button->save();
 
 }
+
+/**
+ * Helper function for granting permissions.
+ */
+function _social_embed_create_permissions() {
+  user_role_grant_permissions('sitemanager', ['administer social embed settings']);
+}
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function social_embed_form_user_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  /** @var \Drupal\Core\Entity\ContentEntityFormInterface $form_object */
+  $form_object = $form_state->getFormObject();
+
+  /** @var \Drupal\social_user\Entity\User $account */
+  $account = $form_object->getEntity();
+
+  /** @var \Drupal\user\UserDataInterface $user_data */
+  $user_data = \Drupal::service('user.data');
+
+  $user_states = $user_data->get('social_embed', $account->id(), 'consent');
+
+  $form['embed_consent'] = [
```

Comment:
I think this fieldset is ok, but I would expect a checkbox to be in place for enabling / disabling the functionality.
If you look at the AC of the story it says:

> by default module is enabled  but settings are disabled so all iframes are loaded as now

> SM can change default settings to show placeholder on site level in place of embedded content

So one checkbox that will act as a feature toggle will be good enough for this story.
A SM can than check that checkbox which is like "By enabling this checkbox users need to give consent to embedded content"